Source/storm/storm.cpp

```diff
@@ -36,8 +37,24 @@ std::string *SBasePath = nullptr;
 
 radon::File &getIni()
 {
-	static radon::File ini(paths::ConfigPath() + "diablo.ini");
-	return ini;
+	if (FileExists("diablo.ini")) {
+		const std::string bdwtpath = paths::BasePath() + "BaseDirWriteTest.foo";

```

Comment:
This should be an `IsWriteableDir` method in file_util

Reply:
Done.